tests/installation/ssh_host_key_settings/verify_ssh_key_not_imported.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: verify that SSH host key dit not got imported.
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    x11_start_program('xterm -geometry 150x45+5+5', target_match => 'xterm');
+    become_root;
+    script_run 'cd /etc/ssh';
+    enter_cmd "cat /etc/ssh/ssh_host_key | tee /dev/$serialdev";
+    wait_serial "SSHHOSTKEYFILE", 5, 1 || die "/etc/ssh/ssh_host_key content does match";
+    enter_cmd "cat /etc/ssh/ssh_host_key.pub | tee /dev/$serialdev";
+    wait_serial "SSHHOSTPUBKEYFILE", 5, 1 || die "/etc/ssh/ssh_host_key.pub content does match";
```

Comment:
that's a neat trick and pretty original for checking string to be honest! I would suggest `assert_matches` to die in case of string mismatch, in combination with a script_output("cat /etc/ssh/ssh_host_key") . However your solution is so cool that I m fine with it as well : D

Reply:
Don't praise me, praise the ones that created https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/c8223c263e0b18c8bafd7fe7345c69d3ae9dbf2e/tests/x11/ssh_key_verify.pm

My contribution here was just to split up that test module into two test modules, so that we're no longer depending on variables like  SSH_KEY_IMPORT or SSH_KEY_DO_NOT_IMPORT. So I didn't look at the internals, just split up the two branches.